xml/article_nfs_storage.xml

```diff
@@ -278,6 +268,16 @@
        </callout>
       </calloutlist>
      </step>
+     <step>
+      <para>
+       Create the file <filename>/etc/drbd.d/nfs-state.res</filename> with the
+       same contents as <filename>/etc/drbd.d/nfs-share.res</filename>, but
+       change the following lines:
+      </para>
+<screen>resource nfs_<emphasis role="bold">state</emphasis> {
+   device /dev/drbd<emphasis role="bold">2</emphasis> minor <emphasis role="bold">2</emphasis>;
+   disk   /dev/nfs/<emphasis role="bold">state</emphasis>;</screen>
+     </step>
```

Comment:
Also the port number should be different than 7790, say use 7791

Reply:
I guess this also means you need to open port 7791 as well in step 1 of `Activating the DRBD devices`?

Reply:
Now pdf is correct. port 7790 is for nfs-share, and port 7791 is for nfs-state.

Reply:
Thanks.  I wasn't fully following this concern before, but seeing the new version, I get it.